src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -716,6 +743,7 @@
   },
   {
     "id": "TA-000028",
+    "name": "SQL.RetentionDays",
```

Comment:
Looking at the rule to confirm, this is on a subset of SQL resources so I think "RetentionDaysWithAudit" to indicate the subset of SQL, and also maybe something that this rule is enforcing a minimum amount of retention time like "MinRetentionDaysWithAudit".

Reply:
Good idea!